pages/posts/the-architectural-scope-of-json-schema.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "The Architectural Scope of JSON Schema"
+date: 2022-07-15
+tags:
+  - Fundamentals
+type: Engineering
+cover: /img/posts/2022/the-architectural-scope-of-json-schema/cover.webp
+authors:
+  - name: Greg Dennis
+    photo: /img/avatars/gregsdennis.webp
+    link: https://twitter.com/gregsdennis
+    byline: JSON Tooling Implementer, Specification & Tooling Architect @ Postman
+excerpt: "What actually is JSON Schema?!"
+---
+
+Recently, I've had a few conversations that got me thinking about JSON Schema's place architecturally.  Today I'd like to share some of those thoughts.
+
+## What _is_ JSON Schema?
+
+JSON Schema is a specification that defines a system of a set of behaviors that can be used to determine whether a JSON value meets certain rules, as well as how those rules can be expressed.
```

Comment:
This seems to focus on validation to the exclusion of annotation.  Given how much you're talking about the implementation/application interface, and that annotations are even more application-oriented than validation (since the application has to decide what to do with the annotations), would you be willing to broaden this a bit?

The validation outcome is a boolean - while implementations may vary in exactly how to access it, in terms of data type and meaning it is unambiguous.  Annotation output (along with errors) is where the work is still in progress.

Reply:
Validation is how JSON Schema started, at least, and most people think of it that way.  However, being the authorities on the subject, it's probably good that we define it how we see it now.

We also discussed elsewhere how validation and annotations are tied to each other.  I don't see validation as separate from annotation; rather evaluation happens and you either get errors or annotations out.